apis/core/v1alpha1/conditions.go

```diff
@@ -32,6 +32,13 @@ const (
 	//		- As a result of InvalidArgument in input yaml
 	//		- Resource server state is "create-failed"
 	ConditionTypeTerminal ConditionType = "ACK.Terminal"
+	// ConditionTypeNonTerminal indicates that the error may be resolved
+	// without needing to update the custom resource spec and sync will continue.
+	// Any error not marked as Terminal will fall in this category.
+	// Examples include:
+	//		- ServiceUnavailable errors that are transient
+	//		- AccessDeniedException that needs correct credentials
+	ConditionTypeNonTerminal ConditionType = "ACK.NonTerminal"
```

Comment:
Can `ACK.NonTerminal` be confusing to users? should it say `ACK.NonTerminalError`/ `ACK.NonTerminalRateLimiting` or something else ?
because anything apart from `ACK.Terminal` is expected to be NonTerminal e.g. `ACK.ResourceSynced`

Reply:
My preference is `ACK.Recoverable`. These aren't necessarily errors, per-se, and what we want to convey to Kubernetes users is that the Condition may be removed through no action of their own.

Reply:
+1 `ACK.Recoverable` 

> we want to convey to Kubernetes users is that the Condition may be removed through no action of their own

not necessarily, action might be required depending on the error but user need not have to resubmit the resource. e.g. 
- `accessDenied` user needs to add some permissions to the role.
- `ThrottlingError/ServiceUnavailable` no action required

Reply:
> Would you still call it `Recoverable` ?

Yes, since the user does not have to change the Spec in order for the Condition to be recovered from.